boot/zephyr/main.c

```diff
@@ -147,6 +141,15 @@ static void do_boot(struct boot_rsp *rsp)
 #endif
 #if CONFIG_MCUBOOT_CLEANUP_ARM_CORE
     cleanup_arm_nvic(); /* cleanup NVIC registers */
+
+#ifdef CONFIG_CPU_CORTEX_M7
+    /* Disable instruction cache and data cache before chain-load the application */
+    SCB_DisableDCache();
+    SCB_DisableICache();
+#endif
+
+#if CONFIG_CPU_HAS_ARM_MPU
+    z_arm_clear_arm_mpu_config();
```

Comment:
can you add `__week` z_arm_clear_arm_mpu_config(void).
Thanks to that mcuboot would be zephyr backward compatible?

Reply:
@nvlsianpu , yes, sure this could be done, hang on... :)